kythe/go/extractors/config/runextractor/mavencmd/mavencmd.go

```diff
@@ -81,7 +82,7 @@ func (m *mavenCommand) Execute(ctx context.Context, fs *flag.FlagSet, args ...in
 		fmt.Sprintf("-Dmaven.compiler.executable=%s", m.javacWrapper)).Run(); err != nil {
 		return m.Fail("error executing maven build: %v", err)
 	}
-	if err := backup.Restore(m.buildFile, tf); err != nil {
+	if err := tf.Restore(); err != nil {
```

Comment:
This is unclear long term.

Short term none of it actually matters because everything gets blown away in a docker image.

Medium term though astroilov mentioned that they might just be passing us a pointer to an already-downloaded repo.

In that world, we would always want to put the config file back into the original form before we started messing with it for preprocessing.

It's unclear long term though, we may end up not needing to restore at all.  We'll see later I think.

Reply:
I see what you mean. Maybe leave a comment to that effect somewhere nearby? (It doesn't need to be every time, just to make it clear "we're cleaning up on purpose regardless whether this succeeds, because blah")

Reply:
sure sounds good